py/private/py_image_layer.bzl

```diff
@@ -0,0 +1,138 @@
+"py_image_layer"
```

Comment:
make the module docstring longer

Also maybe claim that it's unstable, not semver, for 1.0.0

Reply:
docstring seem to be ignored, because of how we do docs here. I just put a big letter experimental in the beginning of the docs.

Reply:
That means the stardoc_with_diff_test points at the wrong file actually.

Reply:
fixed